RubeesEat/Pages/UserHomepage.cshtml

```diff
@@ -3,33 +3,35 @@
 @inject IBillRepository billRepository
 @inject IPersonRepository personRepository
 
-Hello, @personRepository.GetCurrentUser().FirstName!
+<h1 id="hello">Hello, @personRepository.GetCurrentUser().FirstName!</h1>
 
 @{
     var amount = billRepository.GetBalance(personRepository.GetCurrentUser());
     var isPositive = amount >= 0;
-    var displayColor = isPositive ? "green" : "red";
     var sign = isPositive ? "+" : "-";
     var amountAsString = sign + " " + amount.ToString("C", CultureInfo.CreateSpecificCulture("de-DE"));
```

Comment:
I think in the other branches we used the english culture. We should decide which one to use.

Reply:
en puts a $ in front of the number. everywhere else we use en